ai-platform-unified/notebooks/official/featurestore_movie_prediction.ipynb

```diff
@@ -0,0 +1,1277 @@
+        "    entity_id_field=\"movie_id\",\n",
+        "    feature_specs=[\n",
+        "        featurestore_service_pb2.ImportFeatureValuesRequest.FeatureSpec(\n",
+        "            id=\"title\"),\n",
+        "        featurestore_service_pb2.ImportFeatureValuesRequest.FeatureSpec(\n",
+        "            id=\"genres\"),\n",
+        "        featurestore_service_pb2.ImportFeatureValuesRequest.FeatureSpec(\n",
+        "            id=\"average_rating\"),\n",
+        "    ],\n",
+        "    feature_time_field=\"update_time\",\n",
+        "    worker_count=10)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Q-MATtpvm3HI"
+      },
+      "source": [
+        "# Start to import, will take a couple of minutes\n",
+        "ingestion_lro = admin_client.import_feature_values(import_movie_request)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VpaK3yRCnNnI"
+      },
+      "source": [
+        "# Polls for the LRO status and prints when the LRO has completed\n",
+        "ingestion_lro.result()"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9TdxPYdDXjnA"
+      },
+      "source": [
+        "## Online serving"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ezJIMyU-XjnB"
+      },
+      "source": [
+        "The\n",
+        "[Online Serving APIs](https://cloud.google.com/ai-platform-unified/featurestore/docs/reference/rpc/google.cloud.aiplatform.v1beta1#featurestoreonlineservingservice)\n",
+        "lets you serve feature values for small batches of entities. It's designed for latency-sensitive service, such as online model prediction. For example, if you have trained a model to predict if a user is going to watch"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "foNB0D2aw37c"
+      },
+      "source": [
+        "### Read one entity per request\n",
+        "\n",
+        "The ReadFeatureValues API is used to read feature values of one entity; hence\n",
+        "its custom HTTP verb is `readFeatureValues`. By default, the API will return the\n",
+        "latest value of each feature, such as the feature values with the most recent\n",
+        "timestamp.\n",
+        "\n",
+        "To read feature values, specify the entity ID and features to read. The response\n",
+        "contains a `header` and an `entity_view`. Each row of data in the `entity_view`\n",
+        "contains one feature value, in the same order of features as listed in the response header."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3rfWqLrbXjnJ"
+      },
+      "source": [
+        "# Fetch the following 3 features.\n",
```

Comment:
I feel like it might also be useful to show an example of how to unpack the result returned.  E.g. it would be something like 
`result.entity_view.data[2].value.string_array_value.values` to get the list of liked genres... or is there a better way?